examples/muscle_driven_with_contact/contact_forces_inequality_constraint_muscle.py

```diff
@@ -0,0 +1,125 @@
+from matplotlib import pyplot as plt
+import numpy as np
+import biorbd
+
+from biorbd_optim import (
+    Instant,
+    OptimalControlProgram,
+    Constraint,
+    Objective,
+    ProblemType,
+    BidirectionalMapping,
+    Mapping,
+    Bounds,
+    QAndQDotBounds,
+    InitialConditions,
+    ShowResult,
+    Data,
+)
+
+
+def prepare_ocp(model_path, phase_time, number_shooting_points, direction, boundary, show_online_optim=False):
+    # --- Options --- #
+    # Model path
+    biorbd_model = biorbd.Model(model_path)
+    torque_min, torque_max, torque_init = -500, 500, 0
+    activation_min, activation_max, activation_init = 0, 1, 0.5
+    tau_mapping = BidirectionalMapping(Mapping([-1, -1, -1, 0]), Mapping([3]))
+
+    # Add objective functions
+    objective_functions = ({"type": Objective.Mayer.MINIMIZE_PREDICTED_COM_HEIGHT, "weight": -1},)
+
+    # Dynamics
+    problem_type = ProblemType.muscles_and_torque_driven_with_contact
+
+    # Constraints
+    constraints = (
+        {
+            "type": Constraint.CONTACT_FORCE_INEQUALITY,
+            "direction": direction,
+            "instant": Instant.ALL,
+            "contact_force_idx": 1,
+            "boundary": boundary,
+        },
+        {
+            "type": Constraint.CONTACT_FORCE_INEQUALITY,
+            "direction": direction,
+            "instant": Instant.ALL,
+            "contact_force_idx": 2,
+            "boundary": boundary,
+        },
+    )
+
+    # Path constraint
+    nb_q = biorbd_model.nbQ()
+    nb_qdot = nb_q
+    pose_at_first_node = [0, 0, -0.75, 0.75]
+
+    # Initialize X_bounds
+    X_bounds = [QAndQDotBounds(biorbd_model)]
+    X_bounds[0].first_node_min = pose_at_first_node + [0] * nb_qdot
+    X_bounds[0].first_node_max = pose_at_first_node + [0] * nb_qdot
```

Comment:
This will have to be update to in a near future:
```
X_bounds.min[:, 0] = pose_at_first_node + [0] * nb_qdot
X_bounds.max[:, 0] = pose_at_first_node + [0] * nb_qdot
```

Reply:
Done